Booking.cs

```diff
@@ -0,0 +1,206 @@
+﻿using static AirportTicketBooking.Flight;
+namespace AirportTicketBooking
+{
+    internal class Booking
+    {
+        public enum BookedStatus
```

Comment:
create a directory called Enums
and put this enum in a separate class

Reply:
also, ambiguous name, booked states for what?
I suggest renaming it to be `BookingStatus`